templates/controller/name.controller.js

```diff
@@ -1,6 +1,19 @@
 'use strict';
```

Comment:
I don't believe this should be changed. The component type is pretty much just a specific type of directive shorthand. The controller subgenerator *just* creates a controller and has nothing to do with directives.